sys/net/network_layer/fib/universal_address.c

```diff
@@ -201,7 +201,7 @@ int universal_address_compare(universal_address_container_t *entry,
         }
         if( (entry->address[i] & bitmask) == (addr[i] & bitmask) ) {
             ret = entry->address[i] != addr[i];
-            *addr_size_in_bits = (i<<3) + j;
+            *addr_size_in_bits = (i<<3) + (8-j);
```

Comment:
and this caused a faulty size calculation for the matching bits of `entry->address` and  `addr`